pkgs/games/kabeljau/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitea, bash, dialog, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "kabeljau";
+  version = "unstable-2022-04-16";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "papojari";
+    repo = "kabeljau";
+    rev = "c34b6981e508a3049c24a02119240bb110cc1e56";
+    sha256 = "sha256-K7dl6ffkC0Ab29a8ks+NyBHJ67xPkBQ2rhzuhdlv8sk=";
+  };
+
+  buildInputs = [ bash dialog ];
+  nativeBuildInputs = [ makeWrapper ];
+  postPatch = ''
+    patchShebangs --host ${pname}
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${pname}.sh $out/bin/${pname}
+    wrapProgram $out/bin/${pname} --prefix PATH : ${
+      lib.makeBinPath [ bash dialog ]
+    }
+  '';
+
+  meta = with lib; {
+    description = "Survive as a stray cat in an ncurses game";
+    homepage = "https://codeberg.org/papojari/${pname}";
```

```suggestion
    homepage = "https://codeberg.org/papojari/kabeljau";
```

So it can be clicked